Cargo.toml

```diff
@@ -28,10 +28,13 @@ std-fs-io = ["casper-types/std-fs-io"]
 [dependencies]
 async-trait = { version = "0.1.74", optional = true }
 base16 = "0.2.1"
+base64 = "0.21.5"
+bytes = "1.5.0"
```

Comment:
Would it make sense to use capser-types::Bytes instead adding a library ?

Reply:
No, it seems to be a different type.